src/common/post.ts

```diff
@@ -3,7 +3,7 @@ import { Comment, Post } from '../entity';
 import { DataSource } from 'typeorm';
 
 export const defaultImage = {
-  urls: process.env.DEFAULT_IMAGE_URL.split(','),
+  urls: process.env.DEFAULT_IMAGE_URL?.split?.(',') ?? [],
```

Comment:
From time to time, my local test are crashing due to being unable to split. Adding this to avoid the issue.